presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaIntegrationSmokeTest.java

```diff
@@ -149,6 +149,10 @@ public void testRoundTripAllFormats(RoundTripTestCase testCase)
                         "all_datatypes_csv",
                         ImmutableList.of("f_bigint", "f_int", "f_smallint", "f_tinyint", "f_double", "f_boolean", "f_varchar"),
                         ImmutableList.of(100000, 1000, 100, 10, 1000.001, true, "'test'")))
+                .add(new RoundTripTestCase(
```

Comment:
I know that internally `RoundTripTestCase` now sends and receives two rows. But IMO it would be nicer if each row would be different, not same one repeated twice. Can we improve on that?